api/src/main/java/io/strimzi/api/kafka/model/kafka/quotas/QuotasPluginStrimzi.java

```diff
@@ -0,0 +1,149 @@
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model.kafka.quotas;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import io.strimzi.api.kafka.model.common.Constants;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.strimzi.crdgenerator.annotations.Maximum;
+import io.strimzi.crdgenerator.annotations.Minimum;
+import io.sundr.builder.annotations.Buildable;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+import java.util.List;
+
+@Buildable(
+    editableEnabled = false,
+    builderPackage = Constants.FABRIC8_KUBERNETES_API
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({"type", "producerByteRate", "consumerByteRate", "minAvailableBytesPerVolume", "minAvailableRatioPerVolume", "excludedPrincipals"})
+@EqualsAndHashCode(callSuper = true)
+@ToString(callSuper = true)
+public class QuotasPluginStrimzi extends QuotasPlugin {
+    private static final long serialVersionUID = 1L;
+
+    public static final String TYPE_STRIMZI = "strimzi";
+
+    /**
+     * Strimzi Quotas Plugin configuration - admin client configuration prefix
+     */
+    public static final String STRIMZI_ADMIN_CONFIGURATION_PREFIX = "client.quota.callback.static.kafka.admin.";
+
+    /**
+     * Strimzi Quotas Plugin configuration - producer byte rate configuration
+     */
+    public static final String STRIMZI_PRODUCER_BYTE_RATE = "client.quota.callback.static.produce";
+
+    /**
+     * Strimzi Quotas Plugin configuration - consumer byte rate configuration
+     */
+    public static final String STRIMZI_CONSUMER_BYTE_RATE = "client.quota.callback.static.fetch";
+
+    /**
+     * Strimzi Quotas Plugin configuration - minimum available bytes per volume
+     * NOTE: mutual exclusive with {@link #STRIMZI_MIN_AVAILABLE_RATIO_PER_VOLUME}
+     */
+    public static final String STRIMZI_MIN_AVAILABLE_BYTES_PER_VOLUME = "client.quota.callback.static.storage.per.volume.limit.min.available.bytes";
+
+    /**
+     * Strimzi Quotas Plugin configuration - minimum available ratio per volume
+     * NOTE: mutual exclusive with {@link #STRIMZI_MIN_AVAILABLE_BYTES_PER_VOLUME}
+     */
+    public static final String STRIMZI_MIN_AVAILABLE_RATIO_PER_VOLUME = "client.quota.callback.static.storage.per.volume.limit.min.available.ratio";
+
+    /**
+     * Strimzi Quotas Plugin configuration - comma-separated list of excluded principals
+     */
+    public static final String STRIMZI_EXCLUDED_PRINCIPAL_NAME_LIST = "client.quota.callback.static.excluded.principal.name.list";
+
+    /**
+     * Strimzi Quotas Plugin configuration - storage check interval - in seconds
+     */
+    public static final String STRIMZI_CHECK_INTERVAL = "client.quota.callback.static.storage.check-interval";
+
+    public static final String PLUGIN_CLASS_NAME = "io.strimzi.kafka.quotas.StaticQuotaCallback";
```

Comment:
I moved them here from the `QuotasPluginConfiguration` class (which I removed), because the "configuration class" had a different usage than other configuration classes which are in the `cluster-operator` module (where the `QuotasPluginConfiguration` was before). I will check better place for them.